### PR TITLE
CL Banking issue fix

### PR DESCRIPTION
**Pull request by Mister-moon1**
# About the pull request
Hopefully fixes the CLs lack of bank codes along with more simplified code as opposed to the old CL system (why was it like this the div code isnt this important)

# Explain why it's good for the game
CL is the one job that actually has a use for ATMs and being barred out from using them is not fun

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
code: changed the way WY ranks are issued
admin: removed the div code from "select_equipment"
fix: fixed CL job codes
/:cl:

### CODE REVIEW

**Comment by morrowwolf** [comment]
Removing flavor content is pretty lame.

**Comment by Mister-moon1** [comment]
it is also way too complicated for its own good and broke banking for all roles using it

**Review by realforest2001 — COMMENTED** [review]
Minor thing but can you rename it from CL1-10 to WYC1-10? WY Corporate?

**Comment by Mister-moon1** [comment]
yeah sure can do in a bit

File: code/modules/gear_presets/wy.dm [hunk 1]
```diff
@@ -47,66 +47,66 @@
...
-/datum/equipment_preset/wy/manager/New()
-	. = ..()
-	access = get_all_accesses() + get_all_centcom_access()
-
-/datum/equipment_preset/wy/manager/load_gear(mob/living/carbon/human/H)
-	H.equip_to_slot_or_del(new /obj/item/clothing/under/suit_jacket/manager(H), WEAR_BODY)
-	H.equip_to_slot_or_del(new /obj/item/clothing/suit/storage/manager(H), WEAR_JACKET)
-	H.equip_to_slot_or_del(new /obj/item/clothing/head/manager(H), WEAR_HEAD)
-	H.equip_to_slot_or_del(new /obj/item/storage/belt/gun/m4a3/vp78(H), WEAR_WAIST)
-	..()
```
Comment (harryob) [inline code/modules/gear_presets/wy.dm:89]: you removed the parent time for all the /manager subtypes so they won't get the proper equipment, access, id type, skills or headset. it didn't have a `flags` var defines so it didn't show up in game anyway - just set the paygrade to something else

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by harryob — APPROVED** [review]
division code was only ever an admin thing anyway

**Pull request merged**